docs/crds.md

```diff
@@ -1,18 +1,69 @@
-| [ciskubebenchreports][ciskubebenchreports-crd]   | kubebench    | aquasecurity.github.io | false      | CISKubeBenchReport  |
-| [kubehunterreports][kubehunterreports-crd]       | kubehunter   | aquasecurity.github.io | false      | KubeHunterReport    |
+| [vulnerabilityreports][vulnerabilityreports-crd] | vulns,vuln   | aquasecurity.github.io | true       | [VulnerabilityReport](#vulnerabilityreport) |
+| [configauditreports][configauditreports-crd]     | configaudit  | aquasecurity.github.io | true       | [ConfigAuditReport](#configauditreport)     |
+| [ciskubebenchreports][ciskubebenchreports-crd]   | kubebench    | aquasecurity.github.io | false      | [CISKubeBenchReport](#ciskubebenchreport)   |
+| [kubehunterreports][kubehunterreports-crd]       | kubehunter   | aquasecurity.github.io | false      | [KubeHunterReport](#kubehunterreport)       |
+
+## VulnerabilityReport
+
+An instance of the VulnerabilityReport represents the latest vulnerabilities found in a container image of a given
+Kubernetes workload. It consists of a list of OS package and application vulnerabilities with a summary of
+vulnerabilities grouped by severity. For multi-container workloads Starboard creates multiple instances
+of VulnerabilityReports, which are stored in the same namespace and are owned by this workload. Each report follows the
+naming convention `<workload kind>-<workload name>-<container-name>`.
+
+> **NOTE** For various reasons we'll probably change the naming convention to name VulnerabilityReports by image digest (see [#288][issue-288]).
+
+Any static vulnerability scanner that is compliant with the VulnerabilityReport schema can be integrated with Starboard.
+You can find the list of available integrations [here](./integrations/vulnerability-scanners/index.md).
+
+## ConfigAuditReport
+
+An instance of the ConfigAuditReport represents checks performed by configuration auditing tools, such as [Polaris][polaris],
+against a Kubernetes workload's configuration. For example, check that a given container image runs as non root user
+or that a container has resource requests and limits set. Currently checks only relate to Kubernetes workloads, but most
+likely we'll extend this model to cater for other Kubernetes objects such as Services, ConfigMaps, etc (see [#300][discussion-300]).
+
+Each report owned by the underlying Kubernetes workload and is stored in the same namespace, following the
+`<workload-kind>-<workload-name>` naming convention.
+
+Third party Kubernetes configuration checkers, linters, and sanitizers that are compliant with the ConfigAuditReport
+schema can be integrated with Starboard.
+
+> **NOTE** The challenge with onboarding third party configuration checkers is that they tend to have different interfaces
+> to perform scans and vary in output formats for a relatively common goal, which is inspecting deployment descriptors
+> for common configuration pitfalls.
```

Comment:
I think we should add something here about being open to suggestions for changes to the CRD in the case that would enable additional third-party tools

Reply:
I added similar sentence as a note in the top of this page. I think we open for changes to all CRDs, not only ConfigAdutiReport